cli/tests/test_auth.rs

```diff
@@ -568,3 +568,23 @@ fn test_cmd_auth_status_env() {
         ));
     oxide_mock.assert();
 }
+
+#[test]
+fn test_cmd_auth_debug_logging() {
+    let server = MockServer::start();
+    let bad_url = "sys.oxide.invalid";
+
+    // Validate debug logs are printed
+    Command::cargo_bin("oxide")
+        .unwrap()
+        .env("OXIDE_HOST", server.url(""))
+        .env("OXIDE_TOKEN", "oxide-token-bad")
```

Comment:
do we need this for `auth login`?

Reply:
Whoops, good catch. Was trying this test with `auth login` and `auth status` and forgot to remove them.

Fixed in 3c306c0